src/en/dmcscans/src/eu/kanade/tachiyomi/extension/en/dmcscans/DMCScans.kt

```diff
@@ -23,6 +23,12 @@ class DMCScans : ZeistManga("DMC Scans", "https://didascans.blogspot.com", "en")
 
     override val mangaDetailsSelectorGenres = "#labels > a[rel=tag]"
     override val mangaDetailsSelectorInfo = ".imptdt"
+    override val mangaDetailsSelectorDescription = "#desc"
+
+    // =========================== Chapter Feed =============================
+
+    override val chapterFeedRegex = """clwd.\.run\(["'](.*?)["']\)""".toRegex()
+    override val scriptSelector = ".epcheck > script"
```

Comment:
If selectors are constantly updated you need something more robust, like looking through every script tag and a more generic regex, or multiple regexes. Slightly different approach can be seen [in this PR](https://github.com/keiyoushi/extensions-source/pull/2204).